app/connectors/RegistrationConnector.scala

```diff
@@ -38,4 +39,7 @@ class RegistrationConnector @Inject()(config: Configuration, httpClientV2: HttpC
   def getAccounts()(implicit hc: HeaderCarrier): Future[EACDEnrolments] =
     httpClientV2.get(url"$baseUrl/accounts").execute[EACDEnrolments]
 
+  def getVatCustomerInfo()(implicit hc: HeaderCarrier): Future[VatCustomerInfoResponse] =
```

Comment:
we shouldn't need this endpoint as the get registration is a Wrapper of vatInfo and registration and therefore you should be able to get it from doing `request.registrationWrapper.vatInfo` from the request in context https://github.com/hmrc/ioss-returns-frontend/pull/129/files#diff-c9a12a9b7b7afc7e49545de41178ddf64898c92e777c20264d084803342c721aR128

Reply:
Made changes used registrationWrapper.vatInfo

Reply:
Made Changes used registrationWrapper.vatInfo